config/credhub_secret.go

```diff
@@ -0,0 +1,79 @@
+package config
+
+import (
+	"code.cloudfoundry.org/cli/plugin"
+	"errors"
+	"fmt"
+	"github.com/pivotal-cf/spring-cloud-services-cli-plugin/cfutil"
+	"github.com/pivotal-cf/spring-cloud-services-cli-plugin/httpclient"
+	"github.com/pivotal-cf/spring-cloud-services-cli-plugin/serviceutil"
+)
+
+type CredHubSecret interface {
+	Add(string, string, string) error
```

Comment:
Please could you add the parameter names to help make this a bit more descriptive?